docs/tutorials/quickstart-app-auth.md

```diff
@@ -55,6 +53,16 @@ guest. Let's fix that now and add auth.
 
 # The Auth Configuration
 
+Default Backstage installation includes multiple authentication providers out of
+the box. The steps to enable new authentication provider in Backstage are very
+similar to each other, the biggest difference is usually configuring the
+external authentication provider. Please see a subset of possible providers and
+instructions to integrate them below. Steps 1 & 2 are described separately for
+each provider and steps beyond that are common for all.
+
+<details><summary>Github</summary>
+<p>
+
 1. Open `app-config.yaml` and change it as follows
```

Comment:
should all of these numbered items be `###` headings perhaps?

Reply:
Yup, that makes sense. I'll modify accordingly